pyproject.toml

```diff
@@ -30,6 +30,8 @@ classifiers = [
 [tool.poetry.dependencies]
 python = ">=3.9,<3.12"
 space_packet_parser = ">=4.0.2,<5"
+bitstring = '<=4.0.1'
+xarray = '>=2022.11.0'
```

Comment:
I don't think we are using it yet. I realize we will likely later, but I think we should wait to add dependencies until necessary.